dev-tex/rubber/rubber-1.5.1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_5 python3_6 python3_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="A LaTeX wrapper for automatically building documents"
+HOMEPAGE="https://launchpad.net/rubber/"
+SRC_URI="https://launchpad.net/rubber/trunk/${PV}/+download/${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
Please check the license. I see also `GPL-2+` when looking into latex modules, which are shipped insice the archive,

Reply:
Indeed, changed to
```
LICENSE="GPL-2 GPL-2+"